x/pubsub/keeper/keeper.go

```diff
@@ -164,6 +165,28 @@ func (k Keeper) GetSubscribers(ctx sdk.Context) (subscribers []*types.Subscriber
 	return
 }
 
+func (k Keeper) GetValidatorSubscribers(ctx sdk.Context) (subscribers []*types.Subscriber) {
+	allSubscribers := k.GetSubscribers(ctx)
+	for _, subscriber := range allSubscribers {
+		subscriberAddress, err := sdk.AccAddressFromBech32(subscriber.Address)
```

Comment:
I have ptsd from this method now. As long as we're confident it's impossible to get a non-somm prefixed address through `AddSubcriber` this is fine

Reply:
changing it to log an error and continue instead for now